visualizer/dalibo/dalibo.go

```diff
@@ -11,24 +11,29 @@ import (
 // visualizer constants
 const (
 	VisualizerType = "dalibo"
-	postURL        = "https://explain.dalibo.com/new"
+	defaultPostURL = "https://explain.dalibo.com/new"
 	planKey        = "plan"
 )
 
 // Dalibo defines a query plan exporter for the Dalibo visualizer.
 type Dalibo struct {
+	postURL string
 }
 
 // New creates a new Dalibo exporter.
-func New() *Dalibo {
-	return &Dalibo{}
+func New(url string) *Dalibo {
+	if url == "" {
```

Comment:
I suggest renaming the `url` variable (for example, to `postURL`) to avoid a collision with the imported package name `net / url`